src/test/resources/lib/opt/Fail-Safe/jos_01/lib/sysconf/sysconf.la

```diff
@@ -0,0 +1,41 @@
+# sysconf.la - a libtool library file
```

Comment:
built artifact must not be part of the repository. please configure tests to use the built artifact from the target/ directory. please see https://maven.apache.org/plugins-archives/maven-surefire-plugin-2.12.4/examples/configuring-classpath.html for more information